gems/pending/util/postgres_admin.rb

```diff
@@ -255,21 +255,8 @@ def self.runcmd(cmd_str, opts, args)
 
   def self.runcmd_with_logging(cmd_str, opts, params = {})
     $log.info("MIQ(#{name}.#{__method__}) Running command... #{AwesomeSpawn.build_command_line(cmd_str, params)}")
-    with_pgpass_file(opts) do
-      AwesomeSpawn.run!(cmd_str, :params => params).output
-    end
-  end
-
-  # NOTE: keeping .pgpass for external databases. typically will not create/use
-  PGPASS_FILE = '/root/.pgpass'
-  def self.with_pgpass_file(opts)
-    pass = <<-PGPASS
-*:*:#{opts[:dbname]}:#{opts[:username]}:#{opts[:password]}
-*:*:postgres:#{opts[:username]}:#{opts[:password]}
-    PGPASS
-    File.open(PGPASS_FILE, 0600, "w") { |f| f.write pass } if opts[:password]
-    yield
-  ensure
-    File.delete(PGPASS_FILE) if opts[:password]
+    AwesomeSpawn.run!(cmd_str, :params => params, :env => {
+                        "PGUSER"     => opts[:username],
+                        "PGPASSWORD" => opts[:password]}).output
```

Comment:
I'd suggest we change the callers to do the right thing if we're not... specify everything we need to specific.  I believe it's isolated to just a handful of locations.  I don't remember if there was a reason I did a pgpass file instead of environment variables.

Either way, if you are able to do a backup and restore from an appliance using your changes, I think we can fix the fallout in the future.